Autopilot/GPS/Makefile

```diff
@@ -0,0 +1,113 @@
+#
```

Comment:
You can regenerate the makefile any time, by right clicking on the project going to the properties, and then hitting ok.